services/user_service/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /user_service
+
+COPY requirements.txt .
+
+RUN pip3 install -r requirements.txt
+
+COPY . .
```

Comment:
you can move this higher and you won't need to copy requirements separately.

Reply:
agree. Wil add in refactoring commit